docs/build-results-archive.md

```diff
@@ -0,0 +1,52 @@
+# Build results archive.
```

Comment:
Should we specify where this results archive is written to?

Reply:
It is specified in the next section :+1:

Reply:
@Ronoman as mentioned these lines deal with the path and filename conventions: https://github.com/space-ros/process_sarif/pull/6/files#diff-c55023a2efb41f70f1da8ee83deee86689f8bc5e843e745e305ccd8c750c5defR11-R15